validate_user.php

```diff
@@ -0,0 +1,10 @@
+<?php
+// Validate is user logged in
+require_once(__DIR__ . '/validator_functions.php');
+try {
+    validate_user_logged_in();
+    validate_user_is_admin();
```

Comment:
Here you will notice that it validates both the user logged in/exists and they are an admin state. Perhaps decouple these